### DISCUSSION TITLE
RemNote图谱功能要收费，求windows历史版本下载

### DISCUSSION THREAD

**Discussion by treefairyz** [5eagull/RemNote-Chinese-chat-group#30] [你问我答/互相帮助]
RemNote 的最新版本1.3.8里面，图谱功能要收费了，双向链笔记没有图谱功能，怎么能做好知识链接呢？求推荐可以免费用图谱的windows历史版本和下载地址。

**Comment by makerjackie** [comment]
我这只有 1.2.9 版本的 
链接: https://pan.baidu.com/s/1pN95kDc4zqAUSsUqiyjXPw 提取码: v3q3 

1.3.8图谱要收费咩...whatever，我很少用
